ui/public/src/install-ses-lockdown.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import 'ses/dist/ses.umd'; // adds lockdown, harden, and Compartment
+import '@agoric/eventual-send/shim'; // adds support needed by E
+
+// Help lock down the JS environment.  The start compartment (current evaluation context)
+// can still access powerful globals, but this start compartment can use `new Compartment(...)`
+// to evaluate code with stricter confinement.
+lockdown({
+  __allowUnsafeMonkeyPatching__: true,
```

Comment:
Why?

Reply:
It was cargo-culted from a React app.  I will see if I can do without.

Reply:
Yep, looks like it works fine without.  Removing it.